lib/WP/Functions.php

```diff
@@ -84,26 +95,379 @@ function wpEncodeEmoji() {
     return call_user_func_array('wp_encode_emoji', func_get_args());
   }
 
+  function __() {
+    return call_user_func_array('__', func_get_args());
+  }
+
+  function stripslashesDeep() {
+    return call_user_func_array('stripslashes_deep', func_get_args());
+  }
+
+  function wpVerifyNonce() {
+    return call_user_func_array('wp_verify_nonce', func_get_args());
+  }
+
+  function statusHeader() {
+    return call_user_func_array('status_header', func_get_args());
+  }
+
+  function wpJsonEncode() {
+    return call_user_func_array('wp_json_encode', func_get_args());
+  }
+
+  function getObjectTaxonomies() {
+    return call_user_func_array('get_object_taxonomies', func_get_args());
+  }
+
+  function doShortcode() {
+    return call_user_func_array('do_shortcode', func_get_args());
+  }
+
+  function adminUrl() {
+    return call_user_func_array('admin_url', func_get_args());
+  }
+
+  function siteUrl() {
+    return call_user_func_array('site_url', func_get_args());
+  }
+
+  function getPermalink() {
+    return call_user_func_array('get_permalink', func_get_args());
+  }
+
+  function n() {
+    return call_user_func_array('_n', func_get_args());
+  }
+
+  function isMultisite() {
+    return call_user_func_array('is_multisite', func_get_args());
+  }
+
+  function isPluginActive() {
+    return call_user_func_array('is_plugin_active', func_get_args());
+  }
+
+  function isRtl() {
+    return call_user_func_array('is_rtl', func_get_args());
+  }
+
   function currentUserCan() {
     return call_user_func_array('current_user_can', func_get_args());
   }
 
-  function getImageInfo($id) {
```

Comment:
Ok, now I see where it was removed from. Would be clearer if the move was in a single commit indeed. No need to change it now just noting we should pay more attention to that.

Reply:
And I guess I also understand the move now - it's not a WP function wrapper so it shouldn't have been here, right?

Reply:
Exactly!